src/m365/aad/commands/o365group/o365group-conversation-list.ts

```diff
@@ -0,0 +1,60 @@
+import { Logger } from '../../../../cli';
+import {
+  CommandOption
+} from '../../../../Command';
+import GlobalOptions from '../../../../GlobalOptions';
+import commands from '../../commands';
+import { Conversation } from '@microsoft/microsoft-graph-types';
+import { GraphItemsListCommand } from '../../../base/GraphItemsListCommand';
+import Utils from '../../../../Utils';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  groupId?: string;
+}
+
+class AadO365GroupConversationListCommand extends GraphItemsListCommand<Conversation> {
+  public get name(): string {
+    return commands.O365GROUP_CONVERSATION_LIST;
+  }
+
+  public get description(): string {
+    return 'Retrieve conversations for the specified group';
+  }
+
+  public getTelemetryProperties(args: CommandArgs): any {
+    const telemetryProps: any = super.getTelemetryProperties(args);
+    telemetryProps.groupId = typeof args.options.groupId !== 'undefined';
```

Comment:
The groupId is required. We should remove it from the telemetry

Reply:
Removed the telemetry